### PR TITLE
Add special localization guide for ID documentation.

### PR DESCRIPTION
**Pull request by ariscahyadi**
This PR addressing "Establish Central Glossary" as mentioned here #25025

### CODE REVIEW

File: content/id/docs/contribute/localization_id.md [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+Harap berkomitmen dengan terjemahan kamu dan pada saat kamu mendapatkan komentar
+dari pengulas, silahkan atasi sebaik-baiknya. Kami berharap halaman yang 
+diklaim akan diterjemahkan dalam waktu kurang lebih dua minggu. Jika ternyata 
+kamu tidak dapat berkomitmen lagi, beri tahu para pengulas agar mereka dapat 
+meberikan halaman tersebut ke orang lain.
+
+Beberapa acuan tambahan dalam melakukan translasi silahkan lihat informasi 
+berikut ini:
+
+### Daftara Glosarium Translasi dari tim SIG DOC Indonesia
```
[the remark this answers is not in the document]
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:65]: Thanks @frbimo .. yes it is typo :)
Also, let me know whether it is good or clear enough to start the translation task.

File: content/id/docs/contribute/_index.md [hunk 2]
```diff
@@ -75,5 +75,5 @@ terhadap dokumentasi Kubernetes, tetapi daftar ini dapat membantumu memulainya.
 
 - Untuk berkontribusi ke komunitas Kubernetes melalui forum-forum daring seperti Twitter atau Stack Overflow, atau mengetahui tentang pertemuan komunitas (_meetup_) lokal dan acara-acara Kubernetes, kunjungi [situs komunitas Kubernetes](/community/).
 - Untuk mulai berkontribusi ke pengembangan fitur, baca [_cheatseet_ kontributor](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet).
-
+- Untuk kontribusi khusus ke halaman Bahansa Indonesia, baca [Dokumentasi Khusus Untuk Translasi Bahasa Indonesia](/docs/contribute/localization_id.md)
```
Comment (girikuncoro) [inline content/id/docs/contribute/_index.md:78]: typo
also I would not suggest "Translasi" as it has different meaning https://kbbi.kemdikbud.go.id/entri/translasi
```suggestion
- Untuk kontribusi khusus ke halaman Bahasa Indonesia, baca [Dokumentasi Khusus Untuk Penerjemahan Bahasa Indonesia](/docs/contribute/localization_id.md)
```

File: content/id/docs/contribute/localization_id.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
+---
+title: Dokumentasi Khusus Untuk Translasi Bahasa Indonesia
```
Comment (girikuncoro) [inline content/id/docs/contribute/localization_id.md:2]:
```suggestion
title: Dokumentasi Khusus Untuk Penerjemahan Bahasa Indonesia
```

File: content/id/docs/contribute/localization_id.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+content_type: concept
+---
+
+<!-- overview -->
+
+Panduan khusus untuk bergabung ke komunitas SIG DOC Indonesia dan melakukan
```
Comment (girikuncoro) [inline content/id/docs/contribute/localization_id.md:8]: let's make it consistent with kubernetes community (https://kubernetes.io/docs/contribute/#get-involved-with-sig-docs)
```suggestion
Panduan khusus untuk bergabung ke komunitas SIG Docs Indonesia dan melakukan 
```

File: content/id/docs/contribute/localization_id.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
+---
+title: Dokumentasi Khusus Untuk Translasi Bahasa Indonesia
+content_type: concept
+---
+
+<!-- overview -->
+
+Panduan khusus untuk bergabung ke komunitas SIG DOC Indonesia dan melakukan 
+kontribusi untuk mentranslasikan dokumentasi Kubernetes ke dalam Bahasa
```
Comment (girikuncoro) [inline content/id/docs/contribute/localization_id.md:9]:
```suggestion
kontribusi untuk menerjemahkan dokumentasi Kubernetes ke dalam Bahasa 
```

Comment (girikuncoro) [reply content/id/docs/contribute/localization_id.md:9]: let's not use "translasi" for the rest of the page, same comment for the rest of occurence in this page

File: content/id/docs/contribute/localization_id.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+PR _Wrangler_ yang dipilih setiap dua minggu, keduanya akan diumumkan di Slack 
+_channel_ [#kubernetes-docs-id](https://kubernetes.slack.com/archives/CJ1LUCUHM) 
+dari Komunitas Kubernetes.
+
+## Cara Memulai Translasi
+
+Untuk menerjemahkan satu halaman Bahasa Inggris ke Bahasa Indonesia, lakukan 
+langkah-langkah berikut ini:
+
+* Check halaman _issue_ di GitHub dan pastikan tidak ada orang lain yang sudah
```
Comment (girikuncoro) [inline content/id/docs/contribute/localization_id.md:37]: I was thinking if we can add link to our issue tracker, and update accordingly for each milestone cycle

File: content/id/docs/contribute/localization_id.md [hunk 7]
```diff
@@ -0,0 +1,178 @@
...
+Untuk menerjemahkan satu halaman Bahasa Inggris ke Bahasa Indonesia, lakukan 
+langkah-langkah berikut ini:
+
+* Check halaman _issue_ di GitHub dan pastikan tidak ada orang lain yang sudah 
+mengklaim halaman kamu dalam daftar periksa atau komentar-komentar sebelumnya.
+* Klaim halaman kamu pada _issue_ di GitHub dengan memberikan komentar di bawah 
+dengan nama halaman yang ingin kamu terjemahkan dan ambillah hanya satu halaman 
+dalam satu waktu.
+* _Fork_ [repo ini](https://github.com/kubernetes/website), buat terjemahan 
+kamu, dan kirimkan PR (_pull request_) dengan label `language/id`
```
Comment (girikuncoro) [inline content/id/docs/contribute/localization_id.md:43]: Add example of PR description snippet and prow command to add label (/language id)

File: content/id/docs/contribute/localization_id.md [hunk 8]
```diff
@@ -0,0 +1,178 @@
...
+berikut ini:
+
+### Daftara Glosarium Translasi dari tim SIG DOC Indonesia
+Untuk kata-kata selengkapnya silahkan baca glosariumnya 
+[disini](#glosarium-indonesia)
+
+### KBBI
+Konsultasikan dengan KBBI (Kamus Besar Bahasa Indonesia) 
+[disini](https://kbbi.web.id/) dari 
+[Kemendikbud](https://kbbi.kemdikbud.go.id/).
```
Comment (girikuncoro) [inline content/id/docs/contribute/localization_id.md:72]:
```suggestion
Konsultasikan dengan KBBI (Kamus Besar Bahasa Indonesia) 
[di sini](https://kbbi.web.id/) atau situs resmi 
[Kemendikbud](https://kbbi.kemdikbud.go.id/).
```

File: content/id/docs/contribute/localization_id.md [hunk 9]
```diff
@@ -0,0 +1,178 @@
...
+
+### RSNI Glosarium dari Ivan Lanin
+[RSNI Glosarium](https://github.com/jk8s/sig-docs-id-localization-how-tos/blob/master/resources/RSNI-glossarium.pdf)
```
Comment (girikuncoro) [inline content/id/docs/contribute/localization_id.md:75]: is it possible to commit this PDF to k/website under our ID folder and link it to this instead?

File: content/id/docs/contribute/localization_id.md [hunk 10]
```diff
@@ -0,0 +1,178 @@
...
+dan khusus Kubernetes.
+
+
+## Panduan Penulisan _Source Code_
+
+### Mengikuti kode asli dari dokumentasi bahasa Inggris
+
+Untuk kenyamanan pemeliharaan, ikuti lebar teks asli dalam kode bahasa Inggris.
+Dengan kata lain, jika teks asli ditulis dalam baris yang panjang tanpa putus 
+atu baris, maka teks tersebut ditulis panjang dalam satu baris meskipun dalam
```
Comment (girikuncoro) [inline content/id/docs/contribute/localization_id.md:86]: typo
```suggestion
satu baris, maka teks tersebut ditulis panjang dalam satu baris meskipun dalam 
```

File: content/id/docs/contribute/localization_id.md [hunk 11]
```diff
@@ -0,0 +1,178 @@
...
+Sebagai contoh: _service_, _container_, _node_ , dan lain sebagainya. Kata-kata
+sebaiknya ditranslasikan ke Bahasa Indonesia sebagai contoh _service_ menjadi 
+layanan, _container_ menjadi kontainer.
+
+*Tips* : Biasanya kata-kata yang mengacu ke arti yang lebih umum sudah *tidak* 
+ditulis dalam huruf kapital pada halaman asli bahasa Inggris.
+
+### Panduan untuk "Feature Gate" Kubernetes
+
+Istilah [_functional gate_](https://kubernetes.io/ko/docs/reference/command-line-tools-reference/feature-gates/)
```
Comment (girikuncoro) [inline content/id/docs/contribute/localization_id.md:133]: feature gate or functional gate?

File: content/id/docs/contribute/localization_id.md [hunk 12]
```diff
@@ -0,0 +1,178 @@
...
+update |  | pembaruan |  | The latest update... ; Pembaruan terkini...
+state |  | _state_ |  |
+task |  | _task_ |  |
+certificate |  | sertifikat |  |
+install |  | instalasi | https://kbbi.web.id/instalasi | 
+scale |  | skala |  |
+process | kata kerja | memproses | https://kbbi.web.id/proses | 
+replica | kata benda | replika | https://kbbi.web.id/replika | 
+flag |  | tanda, parameter, argumen |  |
+event |  | _event_ |  |
```
Comment (girikuncoro) [inline content/id/docs/contribute/localization_id.md:178]: let's make this alphabetically sorted

**Review by girikuncoro — CHANGES_REQUESTED** [review]
Great effort, I'm excited for this doc so we can start asking for contributions from wider community. I left few comments, let me know what you think! 🎉

File: content/id/docs/contribute/_index.md [hunk 13]
```diff
@@ -75,5 +75,5 @@ terhadap dokumentasi Kubernetes, tetapi daftar ini dapat membantumu memulainya.
 
 - Untuk berkontribusi ke komunitas Kubernetes melalui forum-forum daring seperti Twitter atau Stack Overflow, atau mengetahui tentang pertemuan komunitas (_meetup_) lokal dan acara-acara Kubernetes, kunjungi [situs komunitas Kubernetes](/community/).
 - Untuk mulai berkontribusi ke pengembangan fitur, baca [_cheatseet_ kontributor](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet).
```
Comment (habibrosyad) [inline content/id/docs/contribute/_index.md:77]:
```suggestion
- Untuk mulai berkontribusi ke pengembangan fitur, baca [_cheatsheet_ kontributor](https://github.com/kubernetes/community/tree/master/contributors/guide/contributor-cheatsheet).
```

File: content/id/docs/contribute/localization_id.md [hunk 14]
```diff
@@ -0,0 +1,178 @@
...
+<!-- body -->
+
+## Manajemen _Milestone_ Tim {#manajemen-milestone-tim}
+
+Secara umum siklus translasi dokumentasi ke Bahasa Indonesia akan dilakukan 
+3 kali dalam setahun (sekitar setiap 4 bulan). Untuk menentukan dan mengevaluasi 
+pencapaian atau _milestone_ dalam kurun waktu tersebut [jadwal rapat daring 
+reguler tim Bahasa Indonesia](https://zoom.us/j/6072809193) dilakukan secara 
+konsisten setiap dua minggu sekali. Dalam [agenda rapat ini](https://docs.google.com/document/d/1Qrj-WUAMA11V6KmcfxJsXcPeWwMbFsyBGV4RGbrSRXY) 
+juga dilakukan pemilihan PR _Wrangler_ untuk dua minggu ke depan. Tugas PR
```
Comment (habibrosyad) [inline content/id/docs/contribute/localization_id.md:21]: Why `_Wrangler_` and not just `_wrangler_`? xref https://github.com/kubernetes/website/wiki/PR-Wranglers

File: content/id/docs/contribute/localization_id.md [hunk 15]
```diff
@@ -0,0 +1,178 @@
...
+mengklaim halaman kamu dalam daftar periksa atau komentar-komentar sebelumnya.
+* Klaim halaman kamu pada _issue_ di GitHub dengan memberikan komentar di bawah 
+dengan nama halaman yang ingin kamu terjemahkan dan ambillah hanya satu halaman 
+dalam satu waktu.
+* _Fork_ [repo ini](https://github.com/kubernetes/website), buat terjemahan 
+kamu, dan kirimkan PR (_pull request_) dengan label `language/id`
+* Setelah dikirim, pengulas akan memberikan komentar dalam beberapa hari, dan 
+tolong untuk menjawab semua komentar. Direkomendasikan juga untuk melakukan 
+[_squash_](https://github.com/wprig/wprig/wiki/How-to-squash-commits) _commit_ 
+kamu dengan pesan _commit_ yang baik.
```
Comment (habibrosyad) [inline content/id/docs/contribute/localization_id.md:47]: Should we link `pesan _commit_ yang baik` with https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#commit-message-guidelines?

File: content/id/docs/contribute/localization_id.md [hunk 16]
```diff
@@ -0,0 +1,178 @@
...
+Untuk kenyamanan pemeliharaan, ikuti lebar teks asli dalam kode bahasa Inggris.
+Dengan kata lain, jika teks asli ditulis dalam baris yang panjang tanpa putus 
+atu baris, maka teks tersebut ditulis panjang dalam satu baris meskipun dalam 
+bahasa Indonesia. Jagalah agar tetap serupa.
+
+### Hapus nama reviewer di kode asli bahasa Inggris
+
+Terkadang _reviewer_ ditentukan di bagian atas kode di teks asli Bahasa Inggris. 
+Secara umum, _reviewer-reviewer_ halaman aslinya akan kesulitan untuk meninjau 
+halaman  dalam bahasa Indonesia, jadi hapus kode yang terkait dengan informasi
```
Comment (habibrosyad) [inline content/id/docs/contribute/localization_id.md:93]:
```suggestion
halaman dalam bahasa Indonesia, jadi hapus kode yang terkait dengan informasi 
```

File: content/id/docs/contribute/_index.md (hunk 2 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/_index.md:78]: Ok!

File: content/id/docs/contribute/localization_id.md (hunk 3 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:2]: OK!

File: content/id/docs/contribute/localization_id.md (hunk 4 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:8]: Ok!

File: content/id/docs/contribute/localization_id.md (hunk 5 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:9]: Agree!

File: content/id/docs/contribute/localization_id.md (hunk 6 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:37]: Done!

File: content/id/docs/contribute/localization_id.md (hunk 7 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:43]: Done!

File: content/id/docs/contribute/localization_id.md (hunk 8 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:72]: Done!

File: content/id/docs/contribute/localization_id.md (hunk 10 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:86]: Thanks!

File: content/id/docs/contribute/localization_id.md (hunk 11 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:133]: feature gate!

File: content/id/docs/contribute/_index.md (hunk 13 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/_index.md:77]: Done!

File: content/id/docs/contribute/localization_id.md (hunk 15 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:47]: OK!

File: content/id/docs/contribute/localization_id.md (hunk 12 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:178]: Done but table width still problem :)

File: content/id/docs/contribute/localization_id.md (hunk 11 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:133]: feature gate :)

File: content/id/docs/contribute/localization_id.md (hunk 9 above)
Comment (ariscahyadi) [reply content/id/docs/contribute/localization_id.md:75]: How about put in the `id/_common-resources/` @girikuncoro?

**Review by irvifa — COMMENTED** [review]
🎉 
/lgtm
/approve

**Pull request merged**